core/src/test/java/greencity/service/impl/NotificationServiceImplTest.java

```diff
@@ -1,114 +1,154 @@
 
-import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.ArgumentMatchers.eq;
-import static org.mockito.Mockito.*;
-
-import greencity.GreenCityApplication;
+import greencity.ModelUtils;
+import greencity.dto.category.CategoryDto;
+import greencity.dto.place.PlaceNotificationDto;
 import greencity.dto.user.PlaceAuthorDto;
 import greencity.entity.Category;
 import greencity.entity.Place;
 import greencity.entity.User;
 import greencity.entity.enums.EmailNotification;
-import greencity.entity.enums.PlaceStatus;
+import greencity.message.SendReportEmailMessage;
 import greencity.repository.PlaceRepo;
 import greencity.repository.UserRepo;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.modelmapper.ModelMapper;
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
 import java.time.LocalDateTime;
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
-import org.junit.Before;
-import org.junit.Ignore;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.mockito.InjectMocks;
-import org.mockito.Mock;
-import org.mockito.junit.MockitoJUnitRunner;
-import org.springframework.amqp.rabbit.core.RabbitTemplate;
-import org.springframework.boot.test.context.SpringBootTest;
 
-@RunWith(MockitoJUnitRunner.class)
-@SpringBootTest(classes = GreenCityApplication.class)
-@Ignore
-public class NotificationServiceImplTest {
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(SpringExtension.class)
+class NotificationServiceImplTest {
+
     @InjectMocks
     private NotificationServiceImpl notificationService;
+
     @Mock
     private PlaceRepo placeRepo;
+
+    @Mock
+    private ModelMapper modelMapper;
+
     @Mock
     private UserRepo userRepo;
+
     @Mock
     private RabbitTemplate rabbitTemplate;
-    private List<Place> testPlaces;
-    private EmailNotification emailNotification;
 
-    @Before
-    public void setUp() {
-        Category testCategory = Category.builder().name("CategoryName").build();
-        Place testPlace1 = Place.builder().name("PlaceName1").category(testCategory).build();
-        Place testPlace2 = Place.builder().name("PlaceName2").category(testCategory).build();
-        testPlaces = Arrays.asList(testPlace1, testPlace2);
-
-        when(placeRepo.findAllByModifiedDateBetweenAndStatus(
-            any(LocalDateTime.class),
-            any(LocalDateTime.class),
-            eq(PlaceStatus.APPROVED)
-        )).thenReturn(testPlaces);
+    private EmailNotification emailNotification;
+    private Category category = ModelUtils.getCategory();
+    private User user = ModelUtils.getUser();
+
+    @BeforeEach
+    void setUp() {
```

Comment:
This method is needed to make some configuration, for example like autowiring of some classes or making additional configurations and not for mocking data. Please, move it to appropriate methods